conf/messages

```diff
@@ -97,7 +97,7 @@ error.no_entry_sa-postcode                                   = Enter the postcod
 
 ## Overseas Tax Identifier Page
 tax-identifier.title                                         = Does the business have an overseas tax identifier?
-tax-identifier.hint                                          = This may help us identify the business. It could be a VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify the business details.
+tax-identifier.line_1                                        = We may use this number to help us identify your business. The tax identifier could be a VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify your business details.
```

Comment:
Guess when you reverted the branch, message got reverted to old message, please check with the message content mentioned in the ticket